WordPressKit/WordPressKit/BlogJetpackSettingsServiceRemote.swift

```diff
@@ -0,0 +1,176 @@
+                throw ResponseError.decodingFailure
+        }
+
+        return RemoteBlogJetpackSettings(monitorEnabled: monitorEnabled,
+                                         blockMaliciousLoginAttempts: blockMaliciousLoginAttempts,
+                                         loginWhiteListedIPAddresses: Set(whitelistedIPs),
+                                         sSOEnabled: sSOEnabled,
+                                         sSOMatchAccountsByEmail: sSOMatchAccountsByEmail,
+                                         sSORequireTwoStepAuthentication: sSORequireTwoStepAuthentication)
+    }
+
+    func remoteJetpackMonitorSettingsFromDictionary(_ dictionary: [String: AnyObject]) throws -> RemoteBlogJetpackMonitorSettings {
+
+        guard let monitorEmailNotifications = dictionary[Keys.monitorEmailNotificationsKey] as? Bool,
+            let monitorPushNotifications = dictionary[Keys.monitorPushNotificationsKey] as? Bool else {
+                throw ResponseError.decodingFailure
+        }
+        
+        return RemoteBlogJetpackMonitorSettings(monitorEmailNotifications: monitorEmailNotifications,
+                                                monitorPushNotifications: monitorPushNotifications)
+    }
+
+    func dictionaryFromJetpackSettings(_ settings: RemoteBlogJetpackSettings) -> [String: AnyObject] {
+
+        return [Keys.monitorEnabledKey: settings.monitorEnabled as AnyObject,
+                Keys.blockMaliciousLoginAttemptsKey: settings.blockMaliciousLoginAttempts as AnyObject,
+                Keys.whiteListedIPAddressesKey: settings.loginWhiteListedIPAddresses as AnyObject,
+                Keys.sSOEnabledKey: settings.sSOEnabled as AnyObject,
+                Keys.sSOMatchAccountsByEmailKey: settings.sSOMatchAccountsByEmail as AnyObject,
+                Keys.sSORequireTwoStepAuthenticationKey: settings.sSORequireTwoStepAuthentication as AnyObject]
+
+    }
+
+    func dictionaryFromJetpackMonitorSettings(_ settings: RemoteBlogJetpackMonitorSettings) -> [String: AnyObject] {
+
+        return [Keys.monitorEmailNotificationsKey: settings.monitorEmailNotifications as AnyObject,
+                Keys.monitorPushNotificationsKey: settings.monitorPushNotifications as AnyObject]
+    }
+}
+
+public extension BlogJetpackSettingsServiceRemote {
+
+    public struct Keys {
+        
+        // RemoteBlogJetpackSettings keys
+        public static let monitorEnabledKey = "monitor"
+        public static let blockMaliciousLoginAttemptsKey  = "protect"
+        public static let whiteListedIPAddressesKey = "jetpack_protect_global_whitelist"
+        public static let whiteListedIPsLocalKey = "local"
+        public static let sSOEnabledKey = "sso"
+        public static let sSOMatchAccountsByEmailKey = "jetpack_sso_match_by_email"
+        public static let sSORequireTwoStepAuthenticationKey = "jetpack_sso_require_two_step"
```

Comment:
These three look really odd to me. If the name must start with lowercase, let's do `ssoEnabledKey` instead.

> Acronyms and initialisms that commonly appear as all upper case in American English should be uniformly up- or down-cased according to case conventions:
>
> ```
> var utf8Bytes: [UTF8.CodeUnit]
> var isRepresentableAsASCII = true
> var userSMTPServer: SecureSMTPServer
> ```

https://swift.org/documentation/api-design-guidelines/#follow-case-conventions